docs/source/reference-core.rst

```diff
@@ -670,14 +670,17 @@ expires::
 
 Note that what matters here is the scopes that were active when
 :func:`open_nursery` was called, *not* the scopes active when
-``start_soon`` is called. So for example, the timeout block below does
-nothing at all::
+``start_soon`` is called. The reason for this is that the ``start_soon`` call immediately exits and the scheduled task is awaited by the nursery. So for example, the timeout block below does
```

Comment:
I would recommend leaving this paragraph as it is, and then adding a more detailed explanation below the snippet (rather than linking to the place where the more detailed explanation can be found).

Also, this is a pretty foundational part of the docs, so I think it's important to be extra careful about consistent terminology: for example, the `start_soon` call *returns* (rather than "exits") and the scheduled task *runs inside* (rather than "is awaited by") the nursery.

Reply:
Yeah, adding an extra paragraph or two below is probably a lot clearer than shoehorning it in like I did now. I will revise the PR sometime tomorrow!

Just a heads up: English is not my first language and I'm not fantastic at writing in general so this might take two or three tries to really get right. With that in mind, don't feel bad for having me revise it a couple times more, it's only to be expected. I also think I checked the box to let maintainers edit the PR directly😉